apps/base/kustomize-sandbox-test.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: sample-kustomize
+  namespace: flux-system
+spec:
+  decryption:
+    provider: sops
+  interval: 1m0s
+  prune: true
+  sourceRef:
+    kind: GitRepository
+    name: flux-config
+    namespace: flux-system
+  patches:
+    - patch: |-
+        apiVersion: helm.toolkit.fluxcd.io/v2beta1
+        kind: HelmRelease
+        metadata:
+          name: test-helmrelease
+        spec:
+          interval: 5m
+          install:
+            remediation:
+              retries: 3
+          upgrade:
+            remediation:
+              retries: 3
+              remediateLastFailure: true
+          values:
+            global:
+              environment: ${KEYVAULT_ENVIRONMENT:-prod}
+              tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+              enableKeyVaults: true
+            java:
+              environment:
+                CLUSTER_NAME: "cft-${ENVIRONMENT}-${CLUSTER}-aks"
```

Name it something else, so its easy to distinguish 
```suggestion
                CLUSTER_NAME_2: "cft-${ENVIRONMENT}-${CLUSTER}-aks"
```